source/channels/favorite-channels.rst

```diff
@@ -4,18 +4,30 @@ Mark channels as favorites
 .. include:: ../_static/badges/allplans-cloud-selfhosted.rst
   :start-after: :nosearch:
 
-Marking channels as a favorite is a great way to organize your channel sidebar by choosing which public channels, private channels, and direct messages are most important to you.
+You can mark public channels, private channels, and direct messages as Favorite so they appear in the Favorite category in the channel sidebar.
+To mark a channel as a **Favorites** one, you need to open the channel:
 
-To mark a channel as a favorite, open the channel:
+.. tabs::
 
-**On desktop:** At the top of the page, select on the star icon next to the channel name.
+    .. tab:: Web/Desktop App
 
-.. image:: ../images/favorite-channel-desktop.png
-    :alt: Mark a channel as a favorite.
-       
-This adds the channel to a **Favorites** list at the top of the channel sidebar, so it's easy to access. To remove a channel from the **Favorites** list, select the star again.
+      At the top of the page, select on the star icon next to the channel name.
+
+      .. image:: ../images/favorite-channel-desktop.png
+      :alt: Mark a channel as a favorite.
```

Comment:
I suspect this needs to be intended since it's an attribute of the image directive